### PR TITLE
Render msgOption only if showReadReceipts is enabled

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes [#16803](https://github.com/vector-im/element-web/issues/16803)

### CODE REVIEW

File: src/components/views/rooms/EventTile.js [hunk 1]
```diff
@@ -988,6 +989,17 @@ export default class EventTile extends React.Component {
...
+        if (this.props.showReadReceipts) {
+            const readAvatars = this.getReadAvatars();
+            msgOption = (
+                <div className="mx_EventTile_msgOption">
+                    { readAvatars }
+                </div>
+            );
+        }
+
+
```
Comment (jryans) [inline src/components/views/rooms/EventTile.js:1002]: Nit: Remove the double blank line

File: res/css/structures/_ScrollPanel.scss [hunk 2]
```diff
@@ -21,6 +21,6 @@ limitations under the License.
         display: flex;
         flex-direction: column;
         justify-content: flex-end;
-        overflow-y: hidden;
```
Comment (jryans) [inline res/css/structures/_ScrollPanel.scss:24]: Why is this change needed...? When testing the PR deploy preview here, I didn't notice any difference if I reverted this line back to just `overflow-y`...

**Review by jryans — CHANGES_REQUESTED** [review]
Seems reasonable overall, just a few tweaks to make. 😄

File: res/css/structures/_ScrollPanel.scss (hunk 2 above)
Comment (SimonBrandner) [reply res/css/structures/_ScrollPanel.scss:24]: If you switch to the IRC layout a horizontal scroll bar appears. I originally did that as a temporary fix and accidentally committed it :sweat_smile: There is probably a better solution - I'll look into it

Screenshot_20210415_175057

Comment (SimonBrandner) [reply res/css/structures/_ScrollPanel.scss:24]: Hmmm, with the line removed everything seems to work fine. Hopefully, it makes sense

**Review by jryans — APPROVED** [review]
Thanks, looks reasonable to me! 😄
